.github/workflows/unassign-issue.yml

```diff
@@ -107,8 +107,7 @@ jobs:
             const staleIssues = await github.paginate(github.rest.issues.listForRepo, {
               owner, repo,
               state: 'open',
-              labels: staleLabel,
-              assignee: '*'
```

Comment:
I modified this to handle the assignee unassigning themselves (while the `Assigned` label remains)

Reply:
can we do this in another PR to avoid mixing things?

Reply:
Okay, that makes sense.